clang/lib/3C/ConstraintBuilder.cpp

```diff
@@ -497,7 +498,7 @@ class ConstraintGenVisitor : public RecursiveASTVisitor<ConstraintGenVisitor> {
       for (auto It = Fields.begin();
            InitIdx < E->getNumInits() && It != Fields.end(); InitIdx++, It++) {
         Expr *InitExpr = E->getInit(InitIdx);
-        CB.constrainLocalAssign(nullptr, *It, InitExpr);
+        CB.constrainLocalAssign(nullptr, *It, InitExpr, Same_to_Same, true);
```

Comment:
What is the fifth parameter for? I'm a little disturbed by the addition of extra/more parameters to this function.